airflow-comparison.md

```diff
@@ -0,0 +1,47 @@
+VDK offers dedicated lineage collection for Trino and Impala.
+VDK can collect lineage about individual queries, as well as entire about Data Jobs - namely job start and end objects and job status - failure or success.
+
+
+
+## Templates
+
+We will compare the two platforms based on two forms of templating: the first is the ability of parametrize job executions at runtime, and the second is the ability to build reusable Data Jobs/DAGs which can then be invoked from inside another job/DAG run.
+
+Apache Airflow offers Jinja templating as a method for parametrizing job executions. Jinja is a powerful templating engine based on Django’s templating. It allows the parameterization of arguments passed to operators using runtime variables, as well as more complex expressions.
+For more information on Jinja templating in Airflow, refer to the official documentation - https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html?highlight=jinja#templating-with-jinja
+
+Versatile Data Kit offers a mechanism for injecting runtime variables into SQL queries using Data Job arguments. However, this mechanism offers nowhere near the same level of complexity as Jinja templating does.
+What makes VDK unique in comparison to Airflow however are Data Job templates. These function essentially as Data Jobs, but can be invoked from inside another Data Job, which allows users to standardise ETL/ELT processes among a large team and/or cut redundancy.
+Versatile Data Kit offers out-of-the-box Data Job templates for loading data into SCD1, SCD2 and Periodic Snapshot Fact tables in a Trino or Impala database. Users can also construct their own templates to fit their unique use case.
+For more information on Versatile Data Kit’s templating capacities, refer to our Community Demo - https://www.youtube.com/watch?v=HIRt4bX4ddk
+
+
+## Ingestion
+
+Data ingestion refers to the ability to take some data and send it to a target location.
+
+Versatile Data Kit offers a built-in mechanism for sending data to a configured destination. It provides an API for asynchronous and thread-safe automatic serialization, packaging and sending of data collected from a PEP249-compliant database cursor, a API response call, or any serializable Python iterable object. Additionally, users can configure the parallelism of and frequency at which data is sent, allowing very high throughput of sending data.
+
+Apache Airflow does not provide any out-of-the-box solution for ingesting data; providing support for this is offloaded onto the user.
+
+
+
+## Summary
+
+To summarize, Versatile Data Kit and Apache Airflow’s differences arise out of the different abstraction levels they attempt to provide the user with. Airflow is much lower level, more generalized, offering a wider set of capabilities, whereas VDK offers more restricted functionality aimed specifically at providing data engineers with higher level abstractions tailored to their needs.
```

Comment:
I like the overall article vibe, and I like the summary opening statement. I think we should add to it native extensibility, due to the built-in features are also implemented leveraging the extensibility mechanism, therefore it's proven to be very usable.

Reply:
Maybe "more restricted" could be transformed to "more opinionated"